spec/market_spec.rb

```diff
@@ -0,0 +1,11 @@
+require_relative './spec_helper'
+require_relative '../far_mar'
```

Comment:
Let's move this line into `spec_helper`. Then, the only boilerplate any of the spec files will need is to require `spec_helper`.